config/routes.rb

```diff
@@ -43,7 +47,7 @@
 
   namespace :api do
     namespace :v1 do
-      resources :questionnaire, only: [:show], param: :key
+      resources :questionnaire, only: [:index, :show, :create], param: :key 
```

Comment:
Waar gaat die index heen? Ik zie er geen code voor staan in questionnaire controller?